README.md

```diff
@@ -75,7 +75,7 @@ To enable this propagator in Quarkus, add the following to your `application.yam
 quarkus:
   [...]
 
-  opentelemetry:
```

Comment:
So I think we need to leave this one, as I believe it aligns with what Quarkus is using already.

Reply:
Apologies, I hadn't meant to change that ; was trying to capture the `Opentelemetry` -> `OpenTelemetry` above.